src/WebJobs.Script/Description/DotNet/FSharp/FSharpCompilationService.cs

```diff
@@ -77,7 +69,6 @@ public ICompilation GetFunctionCompilation(FunctionMetadata functionMetadata)
             var resolverSource = resolverSourceBuilder.ToString();
 
             Script<object> script = CodeAnalysis.CSharp.Scripting.CSharpScript.Create(resolverSource, options: _metadataResolver.CreateScriptOptions(), assemblyLoader: AssemblyLoader.Value);
-            Compilation compilation = script.GetCompilation();
```

Comment:
This dependency has been removed. We get all the references from the script and run them directly against the metadata resolver. This call was not cheap, so I'm happy to get rid of it.